app/controllers/organizations_controller.rb

```diff
@@ -1,17 +1,18 @@
 class OrganizationsController < ApplicationController
   before_action :authenticate_user!
+  load_and_authorize_resource
   layout 'org_admin'
 
   def show
     @org = Organization.friendly.find(params[:id])
     session[:current_campaign] ||= Campaign.current_campaign_id(params[:id])
-    if session[:current_campaign]
```

Comment:
Do we need to set the session at all? You use a session to [persist data between requests](http://guides.rubyonrails.org/action_controller_overview.html#session) but I don't see that session being used anywhere other than the `#show` method in that controller (and the test)

We can probably drop this